server/routes/creatingLicences/handlers/caseload.ts

```diff
@@ -1,22 +1,28 @@
 import { Request, Response } from 'express'
 import LicenceService from '../../../services/licenceService'
+import CommunityService from '../../../services/communityService'
 
 export default class CaseloadRoutes {
-  constructor(private readonly licenceService: LicenceService) {}
+  constructor(private readonly licenceService: LicenceService, private readonly communityService: CommunityService) {}
 
   GET = async (req: Request, res: Response): Promise<void> => {
-    const caseload = [
-      {
-        name: 'Adam Balasaravika',
-        crnNumber: 'X381306',
-        conditionalReleaseDate: '03 August 2022',
-      },
-    ]
+    const { staffIdentifier } = await this.communityService.getStaffDetail(res.locals.user.username)
+    const managedOffenders = await this.communityService.getManagedOffenders(staffIdentifier)
+    const caseload = managedOffenders
+      .filter(offender => offender.currentOm)
+      .map(offender => {
+        return {
+          name: offender.offenderSurname,
+          crnNumber: offender.crnNumber,
+          conditionalReleaseDate: '03 August 2022', // TODO: Get conditionalReleaseDate from nomis??
+        }
```

Comment:
The conditional release date will come from prisoner-offender-search as part of the filtering / joining of lists.